frontend/pages/SoftwarePage/helpers.tsx

```diff
@@ -0,0 +1,256 @@
+/**
+ * TODO: Major restructure of directories
+ * 1. Create /software parent directory similar to /hosts, /queries, /policies...
+ * 2. software/SoftwarePage should be it's own subdirectory that is only for the main software nav
+ * 3. Separate this file into /software/helpers and software/SoftwarePage/helpers
+ * 4. software/SoftwarePage will include it's child tabs: /SoftwareTitles /SoftwareOS and /SoftwareVulnerabilities
+ * 5. Create software/components for components shared across software pages such as SoftwareVppForm and FleetAppDetailsForm
+ */
```

Comment:
Spoke to my EM @mostlikelee about this tech debt that has been building up since we originally added a single "`/SoftwarePage`" directory and since, the UI has evolved  to have tabs on the software page, add software pages with tabs, various software detail pages, etc. We still have a legacy "`/SoftwarePage`" every single software FE code lives in, but since we constantly had multiple people working in it, we never had the time to properly restructure the directory to reflect the evolved UI